testsuite/classlibrary/TestVolume.sc

```diff
@@ -16,12 +16,13 @@ TestVolume : UnitTest {
 		OSCFunc({ |msg|
 			queryReply = msg;
 		},'/g_queryTree.reply', s.addr).oneShot;
-		s.sendMsg("/g_queryTree", 0);
+		s.sendMsg("/g_queryTree", 0, 0);
 		s.sync;
 
 		this.assertEquals(queryReply, correctReply,
```

Comment:
This assert will fail if the volume synth has a nodeID that is different than `1000`. This happens sometimes on my system.